### ISSUE TITLE
Publish new version of the package with the Streamlit API fix

### ISSUE THREAD

**Issue by bcarothers-bla** [facebookresearch/hiplot#286]
Could you please publish a new version with the fix implemented in #274? Thank you!

**Comment by MickeyGT** [comment]
I second this issue, if we don't get a new version, it doesn't build the js component and it is unusable with versions of streamlit later than when they removed the _is_running_with streamlit call.

Thank you!

**Comment by Ben-Yeh** [comment]
Agree, would really be useful to have this integration working again through pip!
